skywalking/plugins/sw_pymongo.py

```diff
@@ -101,8 +101,8 @@ def inject_bulk_write(_Bulk, bulk_op_map):
     _execute = _Bulk.execute
 
     def _sw_execute(this: _Bulk, *args, **kwargs):
-        address = this.collection.database.client.address
-        peer = "%s:%s" % address
+        nodes = this.collection.database.client.nodes
```

Comment:
So when the MongoDB is in single-node mode, the `this.collection.database.client.nodes` will also contain one element?

Reply:
I looked at it and the answer is yes.